GVFS/GVFS.Platform.Windows/ProjFSFilter.cs

```diff
@@ -381,6 +381,11 @@ public bool IsReady(JsonTracer tracer, string enlistmentRoot, out string error)
                 TryAttach(enlistmentRoot, out error);
         }
 
+        public bool TryLoad(ITracer tracer, string enlistmentRoot, out string errorMessage)
+        {
+            return TryAttach(enlistmentRoot, out errorMessage);

```

Comment:
Something feels a bit off to me here when it comes to how we're using calling the platform's `KernelDriver` API from the mount verb.

- On Windows `KernelDriver.IsReady` will call `TryAttach` (and the Window's platform `TryLoad` method will never be called).

- On Mac, if `KernelDriver.IsReady` returns false `KernelDriver.TryLoad` will be called and attempt to load the kext.

I'm wondering if we should remove `TryLoad` from the `KernelDriver` interface and have the Mac's `KernelDriver.IsReady` attempt to load the kext if it's not already loaded (just as the Window's `IsReady` will attempt to attach the filter if it's not already attached).  

(Or alternatively, update Window's `IsReady` to not attach the filter and then call `TryLoad` in a place that it will be hit on both platforms)

Reply:
Yeah this was a little weird when I was coding it up because I originally had everything in the `IsReady`.  Changed it to this because Mac needs to display a message to the user before they get prompted for their password so they have some idea why and doing that in the `IsReady` was not the right place because I was using `Console.WriteLine` but that might not be available if the service or something else besides the CLI.  Windows does not need to do that.

Reply:
I will look at refactoring the code so it is more the same on both platforms.

Reply:
Hmmm.  In looking at this, we have the following message currently output before running the `TryAttach`
```
this.Output.WriteLine("Driver not loaded.  Attempting to load. You may be prompted for sudo password...");
```
Windows will not need that message.  So in thinking about solutions, we could:
1. Add another property to the interface for whether or not the message is shown.  I hate to keep adding to that interface though.
2. Pass the `TextWriter` to the `TryLoad` so that it can output the message.  Windows would do nothing and Mac could output this message.  We would need a `null` check on the `TextWriter` so that if something like the service which might not have an output could still call the method.

What are your thoughts?

Reply:
> I hate to keep adding to that interface though.

I agree.

I think Option 2 sounds like the best option.